src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -21,13 +21,15 @@
 #include "SpellAuraEffects.h"
 #include "forge_of_souls.h"
 
-enum Yells
+#define DATA_SOUL_POWER    1
+
+enum Events
 {
-    SAY_AGGRO           = 0,
-    SAY_SLAY            = 1,
-    SAY_DEATH           = 2,
-    SAY_SOUL_STORM      = 3,
-    SAY_CORRUPT_SOUL    = 4,
+    EVENT_MAGIC_BANE      = 1,
+    EVENT_SHADOW_BOLT     = 2,
+    EVENT_CORRUPT_SOUL    = 3,
+    EVENT_SOULSTORM       = 4,
+    EVENT_FEAR            = 5
```

Comment:
[**Nyeriah**](https://github.com/Nyeriah) added a note [19 hours ago](https://github.com/TrinityCore/TrinityCore/pull/14036#discussion_r24174111 "Feb. 5. 2015, 17:14 CET")
Did you check it in the database to see if these values match the data expected to be used? In the last author's PR it was wrong